Resources/Prototypes/Entities/Objects/Misc/monkeycube.yml

```diff
@@ -35,3 +35,41 @@
   - type: Tag
     tags:
     - MonkeyCube
+
+- type: entity
+  parent: BoxCardboard
+  name: monkey cube box
```

Comment:
maybe add "suffix: Syndie"

Reply:
I added the "Syndicate" suffix to the monkey cube box, wrapped monkey cubes and monkey cubes.